lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1719,6 +1722,43 @@ def test_activate_block_id(self):
         )
         self.assertIn("Activate Block ID: test_block_id", response.content)
 
+    def test_negative_position(self):
+        self._assert_redirected_view_at_position(-5, 1)
+
+    def test_positive_position(self):
+        self._assert_redirected_view_at_position(2, 2)
+
+    def test_large_positive_position(self):
+        self._assert_redirected_view_at_position(8, 1)
+
+    def test_zero_position(self):
+        self._assert_redirected_view_at_position(0, 1)
+
```

Comment:
We saw behavior on small negative position values where it used python's negative indexing.  But we want to make sure all negative values set position to 1.  Add a test case to check position = -1.